man/htmlButton.Rd

```diff
@@ -12,10 +12,11 @@
 \usage{
 htmlButton(htmlButton(children=NULL, id=NULL, n_clicks=NULL, n_clicks_timestamp=NULL,
 key=NULL, role=NULL, autoFocus=NULL, disabled=NULL, form=NULL, formAction=NULL,
-formNoValidate=NULL, name=NULL, type=NULL, value=NULL, accessKey=NULL,
-className=NULL, contentEditable=NULL, contextMenu=NULL, dir=NULL,
-draggable=NULL, hidden=NULL, lang=NULL, spellCheck=NULL, style=NULL,
-tabIndex=NULL, title=NULL, loading_state=NULL, ...))
+formEncType=NULL, formMethod=NULL, formNoValidate=NULL, formTarget=NULL,
+name=NULL, type=NULL, value=NULL, accessKey=NULL, className=NULL,
+contentEditable=NULL, contextMenu=NULL, dir=NULL, draggable=NULL, hidden=NULL,
+lang=NULL, spellCheck=NULL, style=NULL, tabIndex=NULL, title=NULL,
+loading_state=NULL, ...))
```

Comment:
Combined with `deps`, `inst`, etc. changes, doesn't this equate to an actual release/update of the R package (considering that GitHub is currently our distribution mechanism)?